evm/package.json

```diff
@@ -4,15 +4,44 @@
   "license": "MIT",
   "scripts": {
     "build": "truffle build",
-    "test": "truffle test",
-    "lint": "echo \"No linter configured for evm project\" &&  exit 0"
+    "depcheck": "echo 'chainlink' && depcheck --ignore-dirs=build/contracts || true",
+    "lint": "echo \"No linter configured for evm project\" &&  exit 0",
+    "solhint": "solhint ./contracts/**/*.sol",
+    "test": "truffle test"
   },
   "dependencies": {
+    "chainlink": "0.6.1",
```

Comment:
Don't think this should be here, the package itself is `chainlink` so we're creating a circular dep.